res/res_pjsip_outbound_authenticator_digest.c

```diff
@@ -317,6 +317,7 @@ static pj_status_t set_outbound_authentication_credentials(pjsip_auth_clt_sess *
 		case AST_SIP_AUTH_TYPE_MD5:
 			pj_cstr(&auth_cred.data, auth->md5_creds);
 			auth_cred.data_type = PJSIP_CRED_DATA_DIGEST;
+			auth_cred.algorithm_type = PJSIP_AUTH_ALGORITHM_MD5;
```

Comment:
Do this need to be guarded with `#ifdef`s for non-bundled builds?

Reply:
Same.